solutions/usaco-1185.mdx

```diff
@@ -0,0 +1,141 @@
+---
+id: usaco-1185
+source: USACO Gold 2022 January
+title: Drought
+author: Ryan Chou
+---
+
+[Official Analysis (C++)](http://usaco.org/current/data/sol_prob1_gold_jan22.html)
+
+## Explanation
+
+Let's consider a case where we have just two cows.
+
+$[2, 2]$
+
+In any case where they're equal, it's always possible to reduce them both to zero! (by decreasing adjacent elements).
+
+However, with three cows:
+
+$[2, 2, 2]$
+
+We can't apply the same transformation. In fact, for any odd number of cows, it's impossible to guarantee that we can decrement all the elements to zero.
```

Comment:
```suggestion
We can't apply the same transformation. In fact, for any odd number of cows, it's impossible to guarantee that we can decrement all the elements to zero even if all the elements are equal.
```
or some other modifier? since "it's impossible to guarantee that we can decrement all the elements to zero" is still true for even numbers

Reply:
intended to say that we can't assure for input where there are an odd number of cows -- does it not come across as that?

Reply:
hmm i mean that i feel like "In fact, for any odd number of cows, it's impossible to guarantee that we can decrement all the elements to zero." implies that it's not possible to guarantee only for odd numbers of cows implying that it's possible to guarantee for even numbers of cows?

Reply:
but it is?

Reply:
umm isn't it only possible to guarantee that for an even number of cows, the elements can all be made zero if all the elements are the same? not with just any assortment of elements

Reply:
omg i misread, sorry!

yep you're right